src/schrodinger/sketcher/model/mol_model.cpp

```diff
@@ -619,6 +620,31 @@ void MolModel::addAttachmentPoint(const RDGeom::Point3D& coords,
                             WhatChanged::MOLECULE);
 }
 
+// TODO: add bound_to_atom param
+void MolModel::addMonomer(const std::string_view res_name,
+                          const rdkit_extensions::ChainType chain_type,
+                          const RDGeom::Point3D& coords)
+{
+    // we'll renumber the chains in assignChains, so for now we just need
+    // something with the correct prefix and a unique number
+    auto chain_id = rdkit_extensions::toString(chain_type) + "999999";
+    auto create_atom = [res_name, chain_id]() {
+        auto monomer_unique_ptr =
+            rdkit_extensions::makeMonomer(res_name, chain_id, 1, false);
+        std::shared_ptr<RDKit::Atom> monomer;
+        monomer.reset(monomer_unique_ptr.release());
```

Comment:
I think its ok to update makeMonomer to return a shared_ptr

Reply:
Cool, I make that change.